### DISCUSSION TITLE
Is v2.9.93-beta a production-ready version?

### DISCUSSION THREAD

**Discussion by ExplodingDragon** [semaphoreui/semaphore#2036] [Q&A] [answered]
Is v2.9.93-beta a production-ready version? He seems to be inconsistent in naming previous versions. @fiftin

**Comment by tboerger** [comment]
I think a suffix like beta is pretty obvious, isn't it? It is BETA, not PRODUCTION.

**Reply by fiftin** [reply, marked as answer]
@ExplodingDragon it is my mistake, I marked Beta as latest release, but not pre-release. Fixed.
